mycroft/stt/__init__.py

```diff
@@ -106,12 +108,31 @@ def execute(self, audio, language=None):
         return self.api.stt(audio.get_flac_data(), self.lang, 1)[0]
 
 
+class KaldiSTT(STT):
+    def __init__(self):
+        super(KaldiSTT, self).__init__()
+
+    def execute(self, audio, language=None):
+        language = language or self.lang
+        config = ConfigurationManager.get().get("stt", {})
+        response = requests.post(self.config.get("uri"), data=audio.get_wav_data())
```

Comment:
also: the module requests need to be imported at the top of the file. You can do

```python
from requests import post
```

And then shorten this line by just making it
`response = post(self.config.get("uri"), data=audio.get_wav_data())`